b+tree.ts

```diff
@@ -488,6 +488,213 @@ export default class BTree<K=any, V=any> implements ISortedMapF<K,V>, ISortedMap
     return {nodequeue, nodeindex, leaf:nextnode};
   }
 
+  /**
+   * Computes the delta operations (adds, deletes, and modifications) required to go from this tree to the supplied destination.
+   * For efficiency, the delta is returned via invocations of supplied handlers.
+   * The computation is optimized for the case in which the two trees have large amounts of shared data (obtained by calling the
+   * `clone` or `with` APIs) and will avoid any iteration of shared state.
+   * @param destination The tree to compute a delta to.
+   * @param processAdd Callback invoked for all adds in the delta.
+   * @param processDelete Callback invoked for all deletions in the delta.
+   * @param processModify Callback invoked for all modifications in the delta.
+   */
+  delta(destination: BTree<K, V>, processAdd: (k: K, v: V) => void, processDelete: (k: K, v: V) => void, processModify: (k: K, vOld: V, vNew: V) => void): void {
```

Comment:
I think 'after' might be a better name than 'destination' (Ex: it is the state of the tree after applying the detected modifications).

I wonder if a static 'diff' taking 'before' and 'after' would be more clear.